json/ChipExplain_BoostSkillExplain.txt

```diff
@@ -368,14 +368,14 @@
 		"jp_explainShort": "発動率アップ",
 		"tr_explainShort": "Boost Activation Rate",
 		"jp_explainLong": "装備したサポートチップの発動率が 10％ 上昇する。",
-		"tr_explainLong": "Support Chip activation rate boosted by 10%."
+		"tr_explainLong": "Boosts linked Support Chip's\\nactivation rate by 10%."
 	},
```

Comment:
I see `\\n` in multiple lines

Reply:
\\n is how it's done in the jp text, and it works fine in testing, what's the issue with it?